sweep/tx_input_set.go

```diff
@@ -131,6 +164,13 @@ func (t *txInputSet) addToState(inp input.Input, constraints addConstraints) *tx
 		return nil
 	}
 
+	// If the input comes with a required tx out that is below dust, we
+	// won't add it.
+	reqOut := inp.RequiredTxOut()
+	if reqOut != nil && btcutil.Amount(reqOut.Value) < t.dustLimit {
```

Comment:
As mentioned above this _shouldn't_ happen since the 2 of 2's _must_ be above the dust limit in order to be considered, but good defensive clause to have nevertheless.